elements/order_slip.php

```diff
@@ -123,11 +123,11 @@
         </p>
 
         <p>
-            <strong><?= t("Payment Method") ?>: </strong><?= $order->getPaymentMethodName() ?><br>
+            <strong><?= t("Payment Method") ?>: </strong><?= t($order->getPaymentMethodName()) ?><br>
```

Comment:
My only concern with this bit here is that the payment method name for each payment method is configurable in the dashboard - these would have already been translated when initially populated.